### PR TITLE
NR-110985: opentelemetry collector docker docs

### PR DESCRIPTION
**Pull request by sigilioso**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

Currently, this is a work in progress, the description will be updated when it is ready to be reviewed.

### CODE REVIEW

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-docker.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+Complete the steps below to collect metrics from docker containers:
+
+## Step 1: Sign up for your free account if you haven't already [#signup]
+
+<InlineSignup />
+
+## Step 2: Prerequisites [#prerequisites]
+
+The collector will use the [dockerstats receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/dockerstatsreceiver) component and it only supports Linux.
```
Comment (mangulonr) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-docker.mdx:25]: Why we support only Linux?
Is this a limitation of the future NRDOT LP?

Comment (sigilioso) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-docker.mdx:25]: It is not only of the NRDOT LP but also of the [docker stats receiver itself](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/dockerstatsreceiver#readme). This shouldn't be an issue as we have the same limitation for [nri-docker integration](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/linux-installation/container-infrastructure-monitoring/#requirements)

Comment (mangulonr) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-docker.mdx:25]: Do you think it is worth mentioning the minimum Docker API version supported (v1.22)?
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/dockerstatsreceiver

Comment (sigilioso) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-docker.mdx]: The minimum API version is likely changing, consider [this PR](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/21803/files) for instance. Taking it into account, I've just rephrased the requirements paragraph here [48002ac](https://github.com/newrelic/docs-website/pull/13190/commits/48002acb3d09af705201a47889a9f013c3896db2), let me know what do you think.

**Review by mangulonr — APPROVED** [review]

**Review by nbaenam — APPROVED** [review]
Thanks for adding this! I've added some minor TW edits

**Pull request merged**